TRB_label_assist/SVM_decision_boundaries.ipynb

```diff
@@ -264,9 +270,9 @@
     ],
     "metadata": {
         "kernelspec": {
-            "display_name": "Python 3",
+            "display_name": "emission-private-eval",
```

Comment:
```suggestion
            "display_name": "Python 3",
```

Reply:
This still shows as a change

Reply:
Reverted this back to Python 3 using git file history.